pegasus/src/homepage.rb

```diff
@@ -184,21 +184,28 @@ def self.get_actions(request)
         }
       ]
     elsif ["post-hoc", "pre-hoc"].include? hoc_mode
-      [
+      actions = [
         {
           text: "homepage_action_text_learn",
           type: "cta_button",
           url: CDO.studio_url("/courses"),
-        },
-        {
-          text: "homepage_action_text_codevideo",
-          type: "video",
-          youtube_id: youtube_id,
-          download_path: download_path,
-          facebook: facebook,
-          twitter: twitter
         }
       ]
+
+      code_video_action = {
+        text: "homepage_action_text_codevideo",
+        type: "video",
+        youtube_id: youtube_id,
+        download_path: download_path,
+        facebook: facebook,
+        twitter: twitter
+      }
+
+      bhm_action = {
+        type: "celebrate_bhm",
```

Comment:
(minor) It seems a little weird to me that this `type` is so specific when the other types are a bit more generic, but up to you.

Reply:
I see what you mean, I could make it more generic.